src/components/DateRangePicker/DateRangePicker.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { useRef } from 'react'
+import './DateRangePicker.css'
+import { SearchFilter, dateTuple } from "@pages/Timeline/types";
+import { date } from 'yup';
+
+interface PropsOBJ {
+    dateRange: [dateTuple?, dateTuple?],
+    setRange: (array: [dateTuple, dateTuple]) => void,
```

Comment:
Kind of trivial but for the sake of code naming conventions:

```suggestion
    setDateRange: (array: [dateTuple, dateTuple]) => void,
```